mkosi.conf

```diff
@@ -40,6 +40,9 @@ KernelCommandLine=systemd.crash_shell
                   panic=-1
                   softlockup_panic=1
                   panic_on_warn=1
+                  # These don't ship proper units with [Install] directives so we have to mask them instead.
+                  systemd.mask=isc-dhcp-server.service
```

Comment:
Is isc-dhcp-server is installed by a dependency?? Or, is this sneaked from somewhere?

Reply:
We install it explicitly, I thought it was required by the systemd-networkd test suite? If not we can remove it.

Reply:
Right. The test for networkd requires the DHCP server. I am confused that the commit is for the storage test. If this is intended, feel free to keep the line.

Reply:
Moved to its own commit